Specifications/QIR/Callables.md

```diff
@@ -256,12 +252,12 @@ containing the array of control qubits as the first element and a tuple of
 the remaining arguments as the second tuple.
 
 For instance, if the base callable expects an argument tuple
-`{ %TupleHeader, i64, %Qubit* }`, then the `Controlled` version expects
-`{ %TupleHeader, %Array*, { %TupleHeader, i64, %Qubit* }* }`, and the twice-`Controlled`
+`{ %Tuple, i64, %Qubit* }`, then the `Controlled` version expects
+`{ %Tuple, %Array*, { %Tuple, i64, %Qubit* }* }`, and the twice-`Controlled`
 version expects
-`{ %TupleHeader, %Array*, { %TupleHeader, %Array*, { %TupleHeader, i64, %Qubit* }* }* }`.
+`{ %Tuple, %Array*, { %Tuple, %Array*, { %Tuple, i64, %Qubit* }* }* }`.
 The "ctl" implementation function always expects
-`{ %TupleHeader, %Array*, { %TupleHeader, i64, %Qubit* }* }`.
+`{ %Tuple, %Array*, { %Tuple, i64, %Qubit* }* }`.
 Thus, if the controlled count is greater than 1,
 `__quantum__rt__callable_invoke` needs to disassemble the argument tuple,
 concatenate the control qubit arrays, and form the expected argument tuple.
```

Comment:
**Important.**
_In this particular paragraph_ replacing the `%TupleHeader` with `%Tuple` is a wrong/misleading thing to do.
For clarity I would recommend to remove any mentioning of `%Tuple..` inside of `{ }` in this paragraph, i.e. to replace 
`{ %Tuple, i64, %Qubit* }` with `{ i64, %Qubit* }` and so on.

If anybody feels uncomfortable regarding such a change, then to leave `{ %TupleHeader, i64, %Qubit* }`. Because this reflects the real internals of the tuple. In this case the `%TupleHeader` will need to be defined somewhere earlier.

Reply:
This is also applicable for 2 more paragraphs below.

Reply:
Thanks for catching this! I indeed overlooked this when I did the edits to update the tuple representation.